src/Data/Vec/Properties.agda

```diff
@@ -623,6 +695,11 @@ map-replicate :  ∀ (f : A → B) (x : A) n →
 map-replicate f x zero = refl
 map-replicate f x (suc n) = P.cong (f x ∷_) (map-replicate f x n)
 
+zipWith-replicate : ∀ {n : ℕ} (_⊕_ : A → B → C) (x : A) (y : B) →
+                  zipWith {n = n} _⊕_ (replicate x) (replicate y) ≡ replicate (x ⊕ y)
```

Comment:
I know the functions below violate this (it would be good to change them as well!) but the start of the second line should aligned with the `∀` rather than the `:`

Reply:
Done